src/components/search/concept-modal/tabs/cdes/related-concepts/related-concept-tag.css

```diff
@@ -0,0 +1,28 @@
+.related-concept-tag {
+    margin: 0 !important;
+    cursor: pointer;
+}
+.related-concept-popover-overlay .ant-popover-inner-content {
+    padding: 0;
+}
+.related-concept-popover-overlay .ant-popover-title {
+    padding: 8px 12px;
+    /* text-align: center; */
+    text-transform: capitalize;
+}
+.related-concept-options {
+    padding-bottom: 4px !important;
+}
+.related-concept-options > .ant-menu-item {
+    margin: 0 !important;
```

Comment:
are we missing something with antd? all these `!important`s don't feel right

Reply:
Good catch. This part is a little hacky. Basically, what we're going for here is the appearance of a dropdown menu without using a dropdown (this uses a `Popover` instead, very similar components though).

antd's `Dropdown` component expects the `overlay` prop to be a `Menu`. It then takes the provided menu and styles it accordingly to give the distinct dropdown menu look. On the other hand, the `Popover` component can take anything as its content and has somewhat different styling.
- You lose the dropdown menu look when using a Popover, so you have to manually override the styling of `Menu` with `important!` rules (although you could probably drop these if more specific selectors were specified here). 

The reason this is using a Popover, even though it is literally mimicking a Dropdown is because:
1. Dropdown doesn't support `right` alignment. Technically, you can still use it (since it has a shared API with Popover) but you lose the overlay arrow since it isn't intended to be used.
2. A dropdown doesn't also have a title. It is *just* a menu. Not really a deal breaker but an added perk of using a popover.